src/tslint.ts

```diff
@@ -86,6 +86,21 @@ class Linter {
     private containsRule(rules: Lint.RuleFailure[], rule: Lint.RuleFailure) {
         return rules.some((r) => r.equals(rule));
     }
+
+    private getRulesDirectories(): Array<string> {
+        const rulesDirectories: Array<string> = [];
+
+        if (this.options.rulesDirectory) {
```

Comment:
don't use implicit type coercion; check for null `this.options.rulesDirectory != null`